jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/SharedClassUtilities.java

```diff
@@ -119,16 +119,20 @@ public class SharedClassUtilities {
 	 * 					If shared classes is disabled for this JVM (that is -Xshareclasses:none is present).
 	 * @throws		IllegalArgumentException
 	 * 					If <code>flags</code> is not a valid value.
+	/*[IF JAVA_SPEC_VERSION < 24]
```

Comment:
`com/ibm/oti/shared/SharedClassesNamedPermission.java` can be removed for JDK24+.
`import com.ibm.oti.shared.SharedClassesNamedPermission.SharedPermissions;` at L28 can be deleted for JDK24+.